components/navigation/nav.css

```diff
@@ -0,0 +1,55 @@
+.header {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  min-height: 10vh;
+  width: 100%;
+  background-color: var(--primary-light);
+}
+
+.logo a {
+  font-size: 2rem;
+  font-weight: 600;
+  color: var(--white);
+}
+
+.header-nav {
+  margin-left: auto;
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.header-nav ul {
+  margin-right: 1rem;
+  font-size: 1.5rem;
+  display: flex;
+  gap: 2rem;
+}
+
+.header-nav li {
+  display: inline-block;
+}
+
+.header-nav a {
+  color: var(--white);
+}
+
+.header-nav .fas {
```

Comment:
Instead of using the font-awesome class names directly, you can use a span with a class name like header-icon and then use .header-icon class name. Or use a utility class name directly on the <i> tag

Reply:
Yes. Changed it. Thank you.